query/src/lib.rs

```diff
@@ -68,6 +69,12 @@ impl Variable {
     }
 }
 
+impl fmt::Debug for Variable {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.0)
```

Comment:
I'm happy with this kind of change: "`Debug` should format the output in a programmer-facing, debugging context", and we know implicitly that it's a `Variable`. But I can imagine other developers _not_ being happy about `Variable` and `PlainSymbol` now printing the same!